src/components/parameters.tsx

```diff
@@ -116,7 +116,7 @@ const Parameters: FunctionComponent<
             open={props.showParameters}
             onClose={props.hideParameters}
             aria-labelledby="form-dialog-title"
-            maxWidth={'md'}
+            maxWidth="md"
```

Comment:
can we get rid of this file completely? similarly to gridexplore?

Reply:
we can, will do in another pr to lighten this one